app/app/(music)/playlist/[id]/PlaylistView.tsx

```diff
@@ -38,15 +38,44 @@ const PlaylistView = ({
         playbuttonContext: playlist.uri,
       }}
     />
-    <h4 className="font-black uppercase pb-2">Average Playlist Statistics</h4>
-    <AudioFeatures audioFeatures={averageAudioFeatures} />
-    <div className="divider" />
-    <OffsetTrackList
-      contextUri={playlist.uri}
-      tracks={tracks}
-      showNumber
-      showAlbum
-    />
+    {playlist.tracks.total !== tracks.length && (
+      <div className="alert alert-warning shadow-lg">
```

Comment:
last thing — move this below the divider below the audio features and add a slight py (`py-2` maybe).